django_sirene/importers.py

```diff
@@ -126,172 +139,192 @@ def _prepare_relateds(self, params, row):
         TODO: Create update method to update label of relateds
         """
         # prepare precreate municipality if needed
-        municipality_id = params.get('municipality_id')
+        municipality_id = params.get("municipality_id")
         if municipality_id and municipality_id not in self.db_municipalities_code:
             municipality = Municipality(
-                code=municipality_id,
-                name=row['LIBCOM']
+                code=municipality_id, name=row["libelleCommuneEtablissement"]
             )
             self.relateds_to_create.add(municipality)
             self.db_municipalities_code.add(municipality.code)
 
-        # prepare precreate activities if needed
-        activity_id = params.get('activity_id')
+        # prepare precreate activity if needed
+        # we will get its name in an other way
+        activity_id = params.get("activity_id")
         if activity_id and activity_id not in self.db_activities_code:
-            activity = Activity(
-                code=activity_id,
-                name=row['LIBAPET']
-            )
+            activity = Activity(code=activity_id, name="")
             self.relateds_to_create.add(activity)
-            self.db_activities_code.add(activity.code)
-
-        # prepare precreate legal_status if needed
-        legal_status_id = params.get('legal_status_id')
-        if legal_status_id and legal_status_id not in self.db_legal_statuses_code:
-            legal_status = LegalStatus(
-                code=legal_status_id,
-                name=row['LIBNJ']
-            )
-            self.relateds_to_create.add(legal_status)
-            self.db_legal_statuses_code.add(legal_status.code)
-
-    def _create_relateds(self):
-        """Bulk create in DB of relateds instances
-
-        objs: list of objects to create
-        """
-        # filter by class type
-        filtered = defaultdict(set)
-        while self.relateds_to_create:
-            related_obj = self.relateds_to_create.pop()
-            filtered[related_obj.__class__].add(related_obj)
-
-        for instance, objs in filtered.items():
-            instance.objects.bulk_create(objs)
-
-    def _update_db(self):
-        """Bulk create relateds in first and then Institutions
-        """
-        self._create_relateds()
-        Institution.objects.bulk_update(
-            self.to_update,
-            batch_size=self.db_batch_size
-        )
-        self.to_update = []
+            self.db_municipalities_code.add(activity.code)
```

Comment:
I think this is a copy-paste mistake :
```suggestion
            self.db_activities_code.add(activity.code)
```